app/main/views/services.py

```diff
@@ -187,57 +187,56 @@ def start_new_draft_service(framework_slug, lot_slug):
     )
 
     section = content.get_section(content.get_next_editable_section_id())
+    if section is None:
+        section = content.get_section(content.get_next_section_id(None))
```

Comment:
Ah, but I wonder if it will break the flow here if we make the service name section non-editable (to get rid of the extra link on the summary page).  I think it's worth getting the frameworks change in to see if it still works. (Or at least test the change locally.)